src/openrct2/network/NetworkConnection.cpp

```diff
@@ -19,8 +19,9 @@
 
 using namespace OpenRCT2;
 
-constexpr size_t NETWORK_DISCONNECT_REASON_BUFFER_SIZE = 256;
-constexpr size_t NetworkBufferSize = 1024 * 64; // 64 KiB, maximum packet size.
+static constexpr size_t kNetworkDisconnectReasonBufSize = 256;
+static constexpr size_t kNetworkBufferSize = 1024 * 64; // 64 KiB, maximum packet size.
+static constexpr size_t kNetworkNoDataTimeout = 20;     // Seconds.
```

Comment:
Odd, this is used to define `kTimeoutMs` below, but CI still picks it up as unused.

Reply:
Hm yeah..

Reply:
Oh I think its because those are debug builds.